components/home-screen/landing-page.tsx

```diff
@@ -56,110 +61,15 @@ export const LandingPage = () => {
-                  </div>
-                </div>
-
-                <div className="p-5 flex flex-col flex-grow">
-                  <h3 className="text-sm font-semibold text-light-text dark:text-dark-text">
-                    {t("showcases.description_label")}
-                  </h3>
-                  <p className="text-light-text dark:text-dark-text text-xs">
-                    {showcase?.description}
-                  </p>
-                  <h3 className="text-sm font-semibold text-light-text dark:text-dark-text mt-2">
-                    {t("showcases.description_version")}
-                  </h3>
-                  <p className="text-light-text dark:text-dark-text text-xs">
-                    {"1.0"}
-                  </p>
-                  <div className="mt-4 flex-grow mb-4">
-                    <h4 className="text-sm font-semibold text-light-text dark:text-dark-text">
-                      {t("showcases.character_label")}
-                    </h4>
-                    <div className="mt-2 space-y-3">
-                      {showcase?.personas?.map((persona: any) => (
-                        <div
-                          key={persona.id}
-                          className="border-[1px] border-dark-border dark:border-light-border flex items-center gap-3 p-3 rounded-md"
-                        >
-                          <Image
-                            src={
-                              ensureBase64HasPrefix(
-                                persona.headshotImage?.content
-                              ) || "/assets/NavBar/Joyce.png"
-                            }
-                            alt={persona.name}
-                            width={44}
-                            height={44}
-                            className="rounded-full w-[44px] h-[44px]"
-                          />
-                          <div>
-                            <p className="text-base text-foreground font-semibold">
-                              {persona.name}
-                            </p>
-                            <p className="text-xs text-gray-500 dark:text-gray-400">
-                              {persona.role}
-                            </p>
-                          </div>
-                        </div>
-                      ))}
-                    </div>
-                  </div>
-
-                  <div className="flex gap-4 mt-auto">
-                    <ButtonOutline className="w-1/2">
-                      {t("action.preview_label")}
-                    </ButtonOutline>
-                    <ButtonOutline className="w-1/2">
-                      {t("action.create_copy_label")}
-                    </ButtonOutline>
-                  </div>
-                </div>
-              </div>
-            </Card>
+          {data?.showcases.filter(searchFilter).map((showcase: Showcase) => (
+            <div key={showcase.id}>
+              <ShowcaseCard showcase={showcase} variant="public" />
```

Comment:
key is default form div, you don't need to wrap it in another div
```suggestion
              <ShowcaseCard key={showcase.id} showcase={showcase} variant="public" />
```

Reply:
Ohhh ok